openfe/protocols/openmm_rbfe/equil_rbfe_methods.py

```diff
@@ -570,25 +509,33 @@ def _default_settings(cls) -> RelativeLigandProtocolSettings:
 
         Returns
         -------
-        RelativeLigandProtocolSettings
+        Settings
           a set of default settings
         """
-        return RelativeLigandProtocolSettings(
-            system_settings=SystemSettings(
-                constraints='HBonds'
-            ),
-            topology_settings=TopologySettings(
-                forcefield={'protein': 'amber99sb.xml',
-                            'ligand': 'openff-2.0.0.offxml',
-                            'solvent': 'tip3p.xml'},
+        return settings.Settings(
+            settings_version=1,
+            forcefield_settings=settings.OpenMMSystemGeneratorFFSettings(),
```

Comment:
Why are we setting forcefields here and then again below  under protocol settings but using twto different objects?